### PR TITLE
Update online game article

### PR DESCRIPTION
**Pull request by ZynoXelek**
A small PR to update the ToC of the article with the main objective of making it shorter.

### CODE REVIEW

File: src/content/posts/multiplayer_online_game.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Trying to make an online multiplayer minigame"
 summary: "A simple article to explain how we made a multiplayer online game using python and what we learnt while doing it, from the very basic use of sockets, to the different communication protocols and a bit of optimization."
-date: 2024-03-11T16:39:09-02:00
```
Comment (ctmbl) [inline src/content/posts/multiplayer_online_game.md:4]: Why modify this? Was this a mistake?

**Review by ctmbl — COMMENTED** [review]
Nice pruning in my opinion 😉 

While we're doing an update of this post maybe you could also add these:
- a link to the repo at the very beginning 🥲 this seems obvious but I think that for now there isn't any link except to the readme
- at every code block (or at least at main ones) a permalink to the code like that:

[`server.py`](https://github.com/iScsc/Haunted-Chronicles/blob/eff735a3cd78b7b9f908d8238945b58e7827e43b/server.py#L1078):
```py
def main():
    """Main function launching the parallel threads to manage the different aspects of the server.
    """
    global MAINSOCKET
    ...
    ```
    
    But these are just ideas!

**Comment by ctmbl** [comment]
also @ZynoXelek you should fetch new changes from the repo, rebase your branch and finally force push to this PR's branch. Because a lot of commits are listed here uselessly 😉

File: src/content/posts/multiplayer_online_game.md (hunk 1 above)
Comment (ZynoXelek) [reply src/content/posts/multiplayer_online_game.md:4]: The date that was written was not obtained using the date command previously, so the format was not exactly the same. I thought it was better to change it.

Comment (ctmbl) [reply src/content/posts/multiplayer_online_game.md:4]: that's nice!

**Review by ctmbl — APPROVED** [review]
very nice update!
ready to merge 🚀

**Pull request merged**